packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -135,6 +130,17 @@ cp -a .%{_bindir}/* \
 find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
 sed -ri 'sX.*/usr/bin/ruby|/usr/bin/env ruby.*$X\#\!/usr/bin/%{?scl:%{scl_prefix}}rubyX' .%{_bindir}/foreman-node
 
+# autosign runner, reactor, key file
+mkdir -p %{buildroot}%{salt_proxy_runners_dir}
+mv %{buildroot}%{gem_instdir}/salt/minion_auth/srv/salt/_runners/* \
+   %{buildroot}%{salt_proxy_runners_dir}
+mkdir -p %{buildroot}%{salt_proxy_reactors_dir}
+mv %{buildroot}%{gem_instdir}/salt/minion_auth/foreman_minion_auth.sls \
+   %{buildroot}%{salt_proxy_reactors_dir}/
+mkdir -p %{buildroot}%{salt_state_grains_dir}
+touch %{buildroot}%{salt_state_grains_dir}/autosign_key
+chmod 644 %{buildroot}%{salt_state_grains_dir}/autosign_key
```

Comment:
technically, the `chmod` is not required, as the perms won't be in the package anyways, thanks to `%ghost` (the `touch` *is* required, tho)

Reply:
As I said: https://github.com/theforeman/foreman-packaging/pull/6968#discussion_r780249542 I'm not sure it would be needed. However, it would deal with any umask. That said, I still wonder if it should 644 or 640. If there are secrets in there, it should not be world readable.

Reply:
I agree, 640 should be preferred. I've removed the referenced `chmod` and adapted it in the post section [here](https://github.com/theforeman/foreman-packaging/pull/6968/commits/eee49308522161dcd9a34bf1155f894524f680aa).